.github/workflows/main.yml

```diff
@@ -8,15 +8,26 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: [1.15.x, 1.16.x, 1.17.x]
-        criu_branch: [master, criu-dev]
+        go-version: [1.17.x, 1.18.x]
+        criu_branch: ["", criu-dev]
 
     steps:
 
     - name: checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     - name: install criu
+      if: matrix.criu_branch == ''
+      env:
+        REPO: https://download.opensuse.org/repositories/devel:/tools:/criu/xUbuntu_20.04
+      run: |
+        curl -fSsl $REPO/Release.key | sudo apt-key add -
+        echo "deb $REPO/ /" | sudo tee /etc/apt/sources.list.d/criu.list
+        sudo apt update
+        sudo apt install criu
```

Comment:
@kolyshkin criu is installed by default in the Ubuntu 20.04 GitHub Action virtual environment as dependency Podman.
https://github.com/actions/virtual-environments/blob/21b8cadc/images/linux/scripts/installers/containers.sh